tools/run-tests-zkasm.js

```diff
@@ -21,14 +21,20 @@ async function main() {
     // Get all zkasm files
     const pathZkasm = path.join(process.cwd(), process.argv[2]);
     const files = await getTestFiles(pathZkasm);
-
+    
+    let exit_code = 0;
     // Run all zkasm files
     // eslint-disable-next-line no-restricted-syntax
     console.log(chalk.yellow('--> Start running zkasm files'));
     for (const file of files) {
         if (file.includes('ignore'))
             continue;
-        await runTest(file, cmPols);
+        if (await runTest(file, cmPols) == 1) {
```

Comment:
From this like, it's not really clear what "1" means. We can either document the function `runTest` to describe which values it returns or rename the function to make it more obvious (e.g. `testPasses`).